O365 Security and Compliance Auth Fix (#34896)
demisto/content: Packs/MicrosoftExchangeOnline/Integrations/SecurityAndComplianceV2/SecurityAndComplianceV2_description.md

    1. Fill in the UPN parameter in the integration configuration.
    2. Run the ***o365-sc-auth-start*** command and follow the instructions.
    3. For testing completion of authorization process run the ***o365-sc-auth-test*** command.

## Security and Compliance Integration Changes

### Overview

> In response to Microsoft's deprecation of the App ID, we need to make the following changes to our app registration in Azure:

Reviewer: ```suggestion
In response to Microsoft's deprecation of the App ID, the following changes to app registration in Azure are required:
```
